mappings/net/minecraft/item/ItemPropertyGetter.mapping

```diff
@@ -1,5 +1,5 @@
 CLASS bca net/minecraft/item/ItemPropertyGetter
-	METHOD call (Lbcb;Lbhi;Lair;)F
+	METHOD call call (Lbcb;Lbhi;Lair;)F
```

Comment:
Did enigma glitch here?

Reply:
Seems so, yes. This is a deobfuscated method and it has somehow generated a mapping for it...

Reply:
Is it safe to just edit this line in the mapping file to fix this @modmuss50?

Reply:
Resolved by 9efcc43